.jenkins/ansible/deploy_jenkins.yml

```diff
@@ -0,0 +1,57 @@
+    - hosts: linux-slaves
+      become: yes
+      vars_files:
+         - vars/variables.var
+      tasks:
+         - name: Add repo keys
+           apt_key: url={{item}} state=present
+           with_items:
+                   - https://download.01.org/intel-sgx/sgx_repo/ubuntu/intel-sgx-deb.key
+                   - https://apt.llvm.org/llvm-snapshot.gpg.key
+                   - https://packages.microsoft.com/keys/microsoft.asc
+         - name: Add repositories for openenclave
+           apt_repository: repo={{item}} state=present update_cache=yes
+           with_items:
+                   - deb http://archive.canonical.com/ubuntu hardy partner
+                   - deb http://apt.llvm.org/xenial/ llvm-toolchain-xenial-7 main
+                   - deb [arch=amd64] https://packages.microsoft.com/ubuntu/16.04/prod xenial main
+         - name: Install  prerequisite packages
+           apt: name{{item}} state=present update_cache=yes
+           with_items:
+                   - clang-7
+                   - libssl-dev
+                   - make
+                   - gcc
+                   - gdb
+                   - g++
+                   - pkg-config
+         - name: Download Intel SGX DCAP Driver
+           get_url:
+                   url: https://download.01.org/intel-sgx/dcap-1.0/sgx_linux_x64_driver_dcap_36594a7.bin
+                   dest: /tmp/sgx_linux_x64_driver_dcap_36594a7.bin
+                   mode: 0755 
+         - name: Install the Intel SGX DCAP Driver
+           command: /tmp/sgx_linux_x64_driver_dcap_36594a7.bin
+         - name: Install the Intel and Open Enclave packages
+           apt: name{{item}} state=present update_cache=yes
+           with_items:
+                   - libsgx-enclave-common
+                   - libsgx-enclave-common-dev
+                   - libsgx-dcap-ql 
+                   - libsgx-dcap-ql-dev
+                   - az-dcap-client
+                   - open-enclave 
+         - name: Install jre needed by Jenkins
+           apt: name{{item}} state=present update_cache=yes
+           with_items:
+                   - openjdk-8-jre
```

Comment:
It looks like this needs to install the docker service and add the "jenkins" user to the docker group

Reply:
Hello John, this only installs and adds the slave to the Jenkins master, and we do not use docker on the Jenkins slaves AFAIK. Does this make sense, or I missed something and I should add the docker service and Jenkins user on the Jenkins slave?

Reply:
@johnkord can you please let me know which is the preferred way?

Reply:
@ahapliuc I just checked now. Docker is installed on the Jenkins slaves. Not sure about its purpose, but I think is required. What's the purpose of Docker for Jenkins slaves ? @johnkord

Reply:
@ahapliuc @ionutbalutoiu We use docker in the Jenkinsfile for Open Enclave: https://github.com/Microsoft/openenclave/blob/master/.jenkins/Jenkinsfile

It is used both on the non-ACC VMs and the ACC VMs (though, as of yesterday, the one "non-simulation container" test has been temporarily disabled on the ACC VMs). Take a look at the test script named "simulationTest", which runs its steps inside the "OETOOLS_IMAGE" docker image.

Around half of our tests in CI are run in a container!

Reply:
Ohhh, you're absolutely right! I don't know how I forgot about the tests runs. Lol

Reply:
Added the docker installation part and also made the user jenkins part of the docker group @ionutbalutoiu @johnkord

Reply:
@ahapliuc I'm looking for the docker changes in the commits and I can't find them!

Reply:
@johnkord there should be in place now, committed to a wrong local branch, sorry about that.